flagging_site/config.py

```diff
@@ -15,7 +15,7 @@
 ROOT_DIR = os.path.abspath(os.path.dirname(__file__))
 DATA_STORE = os.path.join(ROOT_DIR, 'data', '_store')
 VAULT_FILE = os.path.join(ROOT_DIR, 'vault.zip')
-
+API_MAX_HOURS = 48
```

Comment:
Don't just add it to the config.py-- actually call it from the app's config. :) You'll need to use Flask's `current_app` object to make it work: `current_app.config['API_MAX_HOURS']`. Also, no need to define it here, you can just define it directly in the `Config` class. We want it to be an app-level configuration for two reasons:

- First and mainly: because we already put all our other config stuff there so we want to be consistent with how we access these vars.
- Second albeit this is a bit esoteric: so we can switch it off and on depending on whether the app is run with a Postgres database, if we ever need to implement the ability to run with/without a database. _This is unlikely_, but I can imagine a situation in the far-off future where the CRWA needs to change hosting services, or at least needs to redeploy the site without me or a volunteer to assist, and for some reason they struggle setting up the Postgres db. (Maybe Heroku goes out of business.) So we can have a config that sets up a database-less version that would use a different max hours. In addition to this situation being unlikely, it's also bad to mess with an API's behavior via stripping functionality from it (i.e. reducing number of hours it supports) once it's deployed. So that wouldn't be an ideal scenario. But given that the CRWA is trying to spend $0 on this project, things like this are just going to be an unfortunate reality. So tldr we do want this to be configurable, even if it's not something you'd normally want to bump down in a production context.